src/Http/Controllers/Support/ResolveController.php

```diff
@@ -140,6 +146,24 @@ private function resolveFactionIDs($ids)
                 return collect([
                     'id' => $faction->factionID,
                     'name' => $faction->factionName,
+                    'category' => 'faction',
+                ]);
+            });
+
+        return $this->cacheIDsAndReturnUnresolvedIDs($names, $ids);
+    }
+
+    private function resolveInternalUniverseIDs($ids)
```

Comment:
Could we get some comments on these functions?

Reply:
There were already comments on what the function does inside the function. However i have added some php_doc on these functions. I hope that is what you wanted?
https://github.com/eveseat/web/pull/236/commits/b4fc41805b9723d82ce1fa9da4dd3bd6cae2256e